compiler/record-minmax/src/RecordMinMax.cpp

```diff
@@ -39,6 +39,35 @@ using DataType = luci_interpreter::DataType;
 namespace
 {
 
+uint32_t numElements(const luci::CircleNode *node)
+{
+  uint32_t num_elements = 1;
+  for (uint32_t i = 0; i < node->rank(); i++)
+    num_elements *= node->dim(i).value();
+
+  assert(num_elements > 0); // FIX_CALLER_UNLESS
```

Comment:
I think removing this assert would be better.
For example, if we want to check additional validation for zero-sized tensor, `numElements` should return 0.
However, with this assert, those validation could not be done :)

Reply:
And meaning of `numElements` if "the number of elements", `numElements(zero-sized node)` should return 0 I think :D